app/routes/content.js

```diff
@@ -6,6 +6,15 @@ import config from 'ember-get-config';
 import loadAll from 'ember-osf/utils/load-relationship';
 import permissions from 'ember-osf/const/permissions';
 
+// Error handling for API
+const handlers = new Map([
+    // format: ['Message detail', 'page']
+    ['Authentication credentials were not provided.', 'forbidden'], // 401
+    ['You do not have permission to perform this action.', 'forbidden'], // 403
```

Comment:
You can just leave it as forbidden to match the API, then. I was mainly bringing this up since the original ticket said to make it Not Found, but I don't have strong feelings either way.